extension/src/test/suite/vscode/quickPick.test.ts

```diff
@@ -200,11 +200,7 @@ suite('Quick Pick Test Suite', () => {
         { label: 'B', value: 2 },
         { label: 'C', value: 3 },
         { label: 'D', value: 4 },
-        { label: 'E', value: 5 },
```

Comment:
Looking at the test, I don't think there is a bug inside the test but there are a lot of commands being run in `selectMultipleQuickPickItems` due to needing to repeatedly loop over the items. Possibly why the test keeps hitting a timeout 🤔 

I've decreased the number of quick pick items and added `2000`ms to the timeout.